src/styles/blocks/card.scss

```
@@ -0,0 +1,67 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__logo {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &_info {
+    font-weight: 500;
+    font-size: 12px;
+    margin: 0 0 4px;
+    line-height: 18px;
+    color: $text-main;
+  }
+
+  &_kod {
```

Change it in html code, as well)
```suggestion
  &__kod {
```